docs/Homebrew-linuxbrew-core-Maintainer-Guide.md

```diff
@@ -194,21 +194,21 @@ Conflicts:
 
 The `merge-homebrew` command will create a pull-request for you, using `hub`.
 
-It is expected that CI checks on the merge commit of the PR will fail.
-This is due to a bug with Azure Pipelines and its handling of merge
-commits. Master branch builds also fail for the same reason. This is
-OK.
-
-Once the PR is approved by other Homebrew developers, you can finalise
-the merge with:
+Continuous integration checks that the pull request passes
```

Comment:
I confused the verb 'check' with the noun 'check' the first time I read it and had to re-read it a few more times before I understood my mistake.

```suggestion
Continuous integration verifies that the pull request passes
```